src/app/openmrs-api/order-resource.service.ts

```diff
@@ -64,6 +102,19 @@ export class OrderResourceService {
     }));
   }
 
+  saveDrugOrder(payload) {
+    if (payload) {
+      console.log('Payload', payload);
+    }
+    const url = this.appSettingsService.getOpenmrsRestbaseurl().trim() + 'order';
+    const headers = new HttpHeaders({ 'Content-Type': 'application/json' });
+    console.log('headers', headers);
+    return this.http.post(url, JSON.stringify(payload), {headers})
+    .map((response: Response) => {
+      return response;
+    });
+  }
+
   private _excludeVoidedOrder(order) {
```

Comment:
Avoid use of underscores as prefixes to members and methods unless you are using it for getter/setter statements.